.github/workflows/ci.yml

```diff
@@ -20,26 +20,43 @@ jobs:
+          - { path: bundler, name: bundler1, ci_node_total: 2, ci_node_index: 1 }
+          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 0 }
+          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 1 }
           - { path: cargo, name: cargo }
           - { path: common, name: common }
-          - { path: composer, name: composer }
+          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 0 }
+          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 1 }
           - { path: docker, name: docker }
           - { path: elm, name: elm }
           - { path: git_submodules, name: git_submodules }
           - { path: github_actions, name: github_actions }
-          - { path: go_modules, name: go_modules }
+          - { path: go_modules, name: go_module, ci_node_total: 2, index: 0 }
+          - { path: go_modules, name: go_module, ci_node_total: 2, index: 1 }
           - { path: gradle, name: gradle }
-          - { path: hex, name: hex }
+          - { path: hex, name: hex, ci_node_total: 2, ci_node_index: 0 }
+          - { path: hex, name: hex, ci_node_total: 2, ci_node_index: 1 }
           - { path: maven, name: maven }
-          - { path: npm_and_yarn, name: npm_and_yarn }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 0 }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 1 }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 2 }
           - { path: nuget, name: nuget }
-          - { path: omnibus, name: omnibus }
-          - { path: python, name: python }
-          - { path: python, name: python_slow }
-          - { path: pub, name: pub }
-          - { path: terraform, name: terraform }
+          - { path: omnibus, name: omnibus, ci_node_total: 1, ci_node_index: 0 }
+          - { path: python, name: python, ci_node_total: 5, ci_node_index: 0 }
+          - { path: python, name: python, ci_node_total: 5, ci_node_index: 1 }
+          - { path: python, name: python, ci_node_total: 5, ci_node_index: 2 }
+          - { path: python, name: python, ci_node_total: 5, ci_node_index: 3 }
+          - { path: python, name: python, ci_node_total: 5, ci_node_index: 4 }
+          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 0 }
+          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 1 }
+          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 2 }
+          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 3 }
+          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 4 }
+          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 0 }
+          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 1 }
+          - { path: terraform, name: terraform, ci_node_total: 1, ci_node_index: 0 }
```

Comment:
Please forgive my YAML syntax ignorance 😇

Is it possible to use space inside these curly braces to align these arguments? Ex:
```yaml
 - { path: pub,       name: pub,       ci_node_total: 2, ci_node_index: 1 }
 - { path: terraform, name: terraform, ci_node_total: 1, ci_node_index: 0 }
```

It'd make it easier for _me_ to read.

Reply:
In VS Code, I tend to just right-click in YAML files and select "format document" and it auto-spaces things... whatever it spits out might be a sensible default? 

Also, I filed:
* https://github.com/dependabot/dependabot-core/issues/5572